boefjes/boefjes/plugins/kat_dns/main.py

```diff
@@ -12,18 +12,38 @@
 from boefjes.job_models import BoefjeMeta
 
 logger = logging.getLogger(__name__)
-DEFAULT_RECORD_TYPES = {"A", "AAAA", "CAA", "CERT", "RP", "SRV", "TXT", "MX", "NS", "CNAME", "DNAME", "SOA"}
+DEFAULT_RECORD_TYPES = {
+    "A",
+    "AAAA",
+    "CAA",
+    "CERT",
+    "RP",
+    "SRV",
+    "TXT",
+    "MX",
+    "NS",
+    "CNAME",
+    "DNAME",
+    "SOA",
+}
+RECORD_TYPES = getenv("RECORD_TYPES", "")
+REMOTE_NS = getenv("REMOTE_NS", "1.1.1.1")
```

Comment:
Moving this here means the `getenv` call is done on import time when the katalogus settings are not available.